### ISSUE TITLE
Could not create new selections in PCA score plots, and could not do the following steps such as statistical analysis.

### ISSUE THREAD

**Issue by meilinghMIPS** [dfci-cccb/mev#1046]

**Comment by apartensky** [comment]
Please describe the problem in more detail.
To make selections in the PCA scatter plot you should be able to drag the crosshair to select the desired points. The selected points are highlighted in red and are listed below the plot. At this point you can use the "Create Selection" button to save the set. 

At which point did you experience a problem? Was there an error message? How many samples are in your dataset? What browser and operating system version are you using?
Thank you

**Comment by meilinghMIPS** [comment]
Hello,

I did all the steps you mentioned in your last email, and it indicated that
it worked well, but finally no selections had been created. However, the
un-normalised data set can be created selections. I also tried the
example "RNASeq
read count file
<http://mev.tm4.org/container/ui/data/raw_count_matrix.rand.primary.counts.tsv>",
the problem was totally same.

[image: Inline images 1]

I used both google and IE browsers, but couldn't work.

Many thanks,
Meiling

Meiling Han       PhD Candidate

Drug Delivery, Disposition and Dynamics
Monash Institute of Pharmaceutical Sciences
Monash University, Parkville Campus
381 Royal Parade, Parkville
Victoria 3052, Australia


Mobile:  +61 (0) 450 530 219

Email:  meiling.han@monash.edu
